backend/tests/test_auth_api.py

```
@@ -2,160 +1,70 @@
-def test_update_order_not_found(client):
-    response = client.put("/orders/9999", json={"price": 123})
-    assert response.status_code == 404
-    assert response.json()["detail"] == "Order not found"
-
-def test_delete_order_not_found(client):
-    response = client.delete("/orders/9999")
-    assert response.status_code == 404
-    assert response.json()["detail"] == "Order not found"
+    data = response.json()
+    assert "access_token" in data
+    assert data["token_type"] == "bearer"
```

Ruff (0.8.2)

Possible hardcoded password assigned to: "token_type"

(S105)